constructor/osxpkg.py

```diff
@@ -94,7 +94,7 @@ def modify_xml(xml_path, info):
         background_path = join(OSX_DIR, 'MacInstaller.png')
 
     if background_path:
-        logger.info("Using background image", background_path)
+        logger.info("Using background image: %s", background_path)
```

Comment:
yeah, me too. Seems like I previously only inspected the linux/windows logs and flake8 seems to not catch it (I am more used to pylint, which seems too be able to catch it... pylint is generally stricter, but also supports different code styles and options so that is hard to find an agreement what to set... flake8 is like the baseline linter, not so strict, default options are sane, extendable via extension)

After I found this one occurence, I inspected the full logs via printing stdout/stderr also in succeeding tests in my fork of the repo (where the CI was not clogged). Those logs are awesome now: you see how constructor runs start in main, continue in e.g. osxpkg, utils, fcp, back to osxpkg until it finishes etc ...